libr/anal/p/anal_arm_cs.c

```diff
@@ -1647,6 +1660,52 @@ PUSH { r4, r5, r6, r7, lr }
 				(insn->detail->arm.op_count - 1) * 4, ARG (0));
 		}
 		break;
+	case ARM_INS_VSTMIA:
+		r_strbuf_setf (&op->esil, "");
+		width = 0;
+		for (i = 1; i < insn->detail->arm.op_count; i++) {
+			r_strbuf_appendf (&op->esil, "%s,%d,%s,+,=[%d],",
+				REG (i), width, ARG (0), REGSIZE32(i));
+			width += REGSIZE32(i);
+		}
+		// increment if writeback
+		if (insn->detail->arm.writeback == true)
```

Comment:
missing braces {}. also == true can be removed because its implicit if you just do if (..writeback) { ..